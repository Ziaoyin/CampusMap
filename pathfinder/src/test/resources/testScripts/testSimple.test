# tests simple graphs
CreateGraph graph
AddNode graph n1
AddNode graph n2

AddEdge graph n1 n2 4
AddEdge graph n1 n2 5

FindPath graph n1 n2