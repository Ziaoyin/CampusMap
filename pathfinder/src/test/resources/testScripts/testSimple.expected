# tests simple graphs
created graph graph
added node n1 to graph
added node n2 to graph

added edge 4.000 from n1 to n2 in graph
added edge 5.000 from n1 to n2 in graph

path from n1 to n2:
n1 to n2 with weight 4.000
total cost: 4.000